android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/RibWorkerBindMonitoringListener.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.core
+
+/**
+ * Reports total Worker Binding duration [RibWorkerThreadingType]
+ */
+interface RibWorkerBindMonitoringListener {
```

Comment:
Since it's a single method interface, we could make it a `fun interface` for idiomatic instantiation

Reply:
Thanks, let me update in a bit